packages/api/amplify_api/pubspec.yaml

```diff
@@ -9,27 +9,45 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
   flutter: ">=3.0.0"
 
+# Helps `pana` since we do not use Flutter plugins for most platforms
+platforms:
+  ios:
+  android:
+  macos:
+  windows:
+  linux:
+  web:
+
 dependencies:
   amplify_api_android: ">=1.0.0-next.0 <1.0.0-next.1"
   amplify_api_ios: ">=1.0.0-next.0 <1.0.0-next.1"
   amplify_core: ">=1.0.0-next.0+1 <1.0.0-next.1"
   amplify_flutter: ">=1.0.0-next.0+2 <1.0.0-next.1"
+  async: ^2.8.2
   aws_common: ">=0.2.3 <0.3.0"
   collection: ^1.15.0
+  connectivity_plus: ^2.3.6
   flutter:
     sdk: flutter
+  json_annotation: ^4.6.0
   meta: ^1.7.0
   plugin_platform_interface: ^2.0.0
+  web_socket_channel: ^2.2.0
```

Comment:
Can you run `aft deps update` and make sure these all align?

Reply:
just ran it and it dit not update anything locally